drivers/project_scorecard/app/controllers/project_scorecard/warehouse_reports/scorecards_controller.rb

```diff
@@ -0,0 +1,162 @@
+###
+# Copyright 2016 - 2020 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module ProjectScorecard::WarehouseReports
+  class ScorecardsController < ApplicationController
+    include WarehouseReportAuthorization
+    include ArelHelper
+    before_action :set_projects, :set_project_groups, :set_current_reports
+    before_action :set_report, only: [:show, :edit, :update]
+
+    def index
+      start_date = Date.current.prev_month.beginning_of_month
+      end_date = Date.current.prev_month.end_of_month
+
+      @range = ::Filters::DateRange.new(start: start_date, end: end_date)
```

Comment:
This should use the new `Filters::FilterBase`.  It should cover the vast majority of cases (feel free to add new ones if there are any, or if necessary extend it.)